tests/perf/workflows/workflow_test.go

```diff
@@ -35,8 +35,8 @@ import (
 )
 
 var tr *runner.TestRunner
-
-var testAppNames = []string{"perf-workflowsapp"}
+var backends = []string{"", "sqlite"}
```

Comment:
I don't think we need to run perf tests for each backend, first and foremost because each test already takes almost 2 hours. If you add 2 backends, the test takes 4 hours. A regular workday has 8 hours, so you see where I'm going :)

Second, the SQLite backend is primarily meant for dev/test and its perf isn't key.

Maybe one option is to make the backend configurable via an env var? By default it would use actors.